docs/netlify/_redirects

```diff
@@ -0,0 +1,534 @@
+/docs/ManualColumnMove.html /docs/javascript-data-grid/api/manual-column-move/
+/docs/ManualColumnResize.html /docs/javascript-data-grid/api/manual-column-resize/
+/docs/ManualRowMove.html /docs/javascript-data-grid/api/manual-row-move/
+/docs/ManualRowResize.html /docs/javascript-data-grid/api/manual-row-resize/
+/docs/Matrix.html /docs/javascript-data-grid/api/matrix/
+/docs/Menu.html /docs/javascript-data-grid/api/menu/
+/docs/MergeCells.html /docs/javascript-data-grid/api/merge-cells/
+/docs/MergeCellsAction.html /docs/javascript-data-grid/api/merge-cells-action/
+/docs/MergedCellCoords.html /docs/javascript-data-grid/api/merged-cell-coords/
+/docs/MergedCellsCollection.html /docs/javascript-data-grid/api/merged-cells-collection/
+/docs/MultiColumnSorting.html /docs/javascript-data-grid/api/multi-column-sorting/
+/docs/MultipleSelectUI.html /docs/javascript-data-grid/multipleselectui
+/docs/NestedHeaders.html /docs/javascript-data-grid/api/nested-headers/
+/docs/NestedRows.html /docs/javascript-data-grid/api/nested-rows/
+/docs/NodeStructure.html /docs/javascript-data-grid/api/node-structure/
+/docs/ObserveChanges.html /docs/javascript-data-grid/api/observe-changes/
+/docs/OperatorsComponent.html /docs/javascript-data-grid/api/operators-component/
+/docs/Options.html /docs/javascript-data-grid/api/options/
+/docs/PersistentState.html /docs/javascript-data-grid/api/persistent-state/
+/docs/Queue.html /docs/javascript-data-grid/api/queue/
+/docs/RadioInputUI.html /docs/javascript-data-grid/radioinputui
+/docs/RecordTranslator.html /docs/javascript-data-grid/api/record-translator/
+/docs/RowsMapper.html /docs/javascript-data-grid/api/rows-mapper/
+/docs/SamplesGenerator.html /docs/javascript-data-grid/api/samples-generator/
+/docs/Search.html /docs/javascript-data-grid/api/search/
+/docs/Selection.html /docs/javascript-data-grid/api/selection/
+/docs/SelectionCalculations.html /docs/javascript-data-grid/api/selection-calculations/
+/docs/SelectionRange.html /docs/javascript-data-grid/api/selection-range/
+/docs/SelectUI.html /docs/javascript-data-grid/selectui
+/docs/Sheet.html /docs/javascript-data-grid/api/sheet/
+/docs/Stack.html /docs/javascript-data-grid/api/stack/
+/docs/Storage.html /docs/javascript-data-grid/api/storage/
+/docs/TableView.html /docs/javascript-data-grid/api/table-view/
+/docs/TextEditor.html /docs/javascript-data-grid/api/text-editor/
+/docs/Transformation.html /docs/javascript-data-grid/api/transformation/
+/docs/TrimRows.html /docs/javascript-data-grid/api/trim-rows/
+/docs/UndoRedo.html /docs/javascript-data-grid/api/undo-redo/
+/docs/UndoRedoSnapshot.html /docs/javascript-data-grid/api/undo-redo-snapshot/
+/docs/UnmergeCellsAction.html /docs/javascript-data-grid/api/unmerge-cells-action/
+/docs/ValueComponent.htm /docs/javascript-data-grid/valuecomponent.htm
+
+/docs/12.1/ActionBarComponent.html /docs/12.1/javascript-data-grid/api/action-bar-component/
```

Comment:
Those redirects should apply to all range of docs versions - not only 12.1, for example:
`https://handsontable.com/docs/10.0/UndoRedo.html -> https://handsontable.com/docs/10.0/api/undo-redo/`
`https://handsontable.com/docs/15.0/UndoRedo.html -> https://handsontable.com/docs/15.0/javascript-data-grid/api/undo-redo/`